pkg/interactor/data.go

```diff
@@ -0,0 +1,55 @@
+package interactor
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/CESARBR/knot-cloud-storage/pkg/data"
+	. "github.com/CESARBR/knot-cloud-storage/pkg/entities"
+)
+
+type DataInteractor struct {
+	DataStore data.DataStore
+}
+
+func NewDataInteractor(dataStore *data.DataStore) *DataInteractor {
+	return &DataInteractor{*dataStore}
+}
```

Comment:
Thanks for the update! Just make sure this follows the language's best practices :point_down: 
```suggestion
type DataInteractor struct {
	DataStore *data.DataStore
}

func NewDataInteractor(dataStore *data.DataStore) *DataInteractor {
	return &DataInteractor{dataStore}
}
```

Reply:
Done!